portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsJumpToPage.testcase

```diff
@@ -53,7 +53,7 @@ definition {
 			formTitleNumber = "1");
 
 		Form.gotoAddField(
-			fieldPositionNumber = "1",
+			fieldPositionNumber = "0",
```

Comment:
@mlan0, is this parameter necessary even with the changes introduced by the first commit?

Reply:
No, it is not necessary, but I think that if I remove all occurrences, it may lose readability. What do you think? Should I remove them? @natocesarrego